core/best-practices/commit-messages.md

```diff
@@ -1,109 +1,262 @@
 
+<!--
 #### Guidelines
+-->
+
+#### ガイドライン
 
+<!--
 *   Must be separated from the summary by a blank line.
 *   Can be multiple paragraphs if necessary, separated by blank lines. It is not unreasonable for a commit message to be more verbose than the final changeset itself.
 *   Unlike the Summary, lines should not be manually wrapped – log viewers can take care of wrapping the description themselves, if they need to.
+-->
 
+*   要約と空白行で区切らなければなりません。
+*   必要であれば、空行で区切って複数の段落にできます。コミットメッセージが最終的な変更セットそのものよりも冗長である場合もあります。
+*   要約とは異なり、行を手動で折り返す必要はありません。必要に応じて、ログビューア自身が説明を折り返すことができます。
+
+<!--
 ### Props
+-->
+
+### Props (称賛)
 
+<!--
 Props should be given to all those who contributed to the final commit, whether through patches, refreshed patches, code suggested otherwise, design, writing, user testing, or other significant investments of time and effort. Usernames are parsed for the credits list and WordPress.org profiles.
+-->
+
+パッチ、更新されたパッチ、提案されたコード、デザイン、ライティング、ユーザーテスト、その他多大な時間と労力を費やして最終的なコミットに貢献したすべての人に賞賛が贈られるべきです。ユーザー名はクレジットリストと WordPress.org プロフィールのために解析されます。
 
+<!--
 In the case of bug reports, props should also be given to the reporter of the bug.
+-->
 
+バグレポートの場合、バグの報告者にも props が与えられるべきです。
+
+<!--
 Check any tickets which were closed as a duplicate in case they contain contributions that warrant props too.
+-->
+
+重複としてクローズされたチケットも、props に値する貢献が含まれている可能性があるためチェックしてください。
 
+<!--
 #### Guidelines
+-->
+
+#### ガイドライン
 
+<!--
 *   Must be preceded by a blank line.
 *   Usernames must not start with an `@` (at) sign.
 *   Separate usernames by comma + space. Think: `/^props (\s*([^,]+),?)+$/`
 *   Copy/paste usernames to avoid typos. (Sorry, rmccue; or is that rmmcue?)
 *   If the user has a space in their displayed name, use the slug from their w.org profile URL. For example `Frank Klein` on Trac should get props as `frank-klein`.
 *   Err on the side of giving props liberally. Props provide major encouragement for contributors.
 *   If you forget to prop someone, check to see if they already have props in the current release as it won’t matter in the long run as they’ll be included in the release credits anyway. If they aren’t already propped, then you can flag it to the [Release Coordinator](https://make.wordpress.org/core/handbook/about/release-cycle/wordpress-release-team-and-focus-leads/#release-co-ordinator) so they can ensure that person is added on release day. It’s also recommended to reach out to the contributor in Slack or in a comment on the ticket as a courtesy and apologize for missing their name in the commit message and letting them know their contribution will be recognized and note how.
+-->
 
+*   先頭には空白行が必要です。
+*   ユーザー名は `@` (アット) 記号で始めることはできません。
+*   ユーザー名はカンマとスペースで区切ってください。正規表現: `/^props (\s*([^,]+),?)+$/`
+*   タイプミスを避けるため、ユーザー名はコピー & ペーストしてください。(ごめんなさい、rmccue さん。それとも rmmcue さんですか ?)
```

Comment:
Copy/paste usernames to avoid typos. (Sorry, rmccue; or is that rmmcue?)
タイプミスを避けるため、ユーザー名はコピー & ペーストしてください。(ごめんなさい、rmccue さん。それとも rmmcue さんですか ?)

---

括弧の文章は削除してしまった方が良いかもしれません

Reply:
内輪受けですね。
https://profiles.wordpress.org/rmccue/

削除しても良いですし、訳すなら「rmccue ごめん。あれ、rmmcue だっけ ?」とか。

Reply:
なるほど。。日本語だと通じにくい気がするので、削除します😅